### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stratkg**

### CODE REVIEW

File: vignettes/Data_Normalization.Rmd [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+```
+
+
+```{r spans plot, out.width = "600px", echo=FALSE}
+knitr::include_graphics("SPANS_pep.png")
+```
+
+
+```{r}
+head(spans_result)
```
Comment (clabornd) [inline vignettes/Data_Normalization.Rmd:327]: This still breaks for me, `spans_result` doesn't exist during vignette building since the chunk that creates it is eval=FALSE.

File: vignettes/RNAseq_Processing_Workflow.Rmd [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+```{r}
+stats_edger <- diffexp_seq(omicsData = myseq, method = "edgeR")
+```
+Various plot types are available to display the results of the statistical comparisons. 
+
+- WHAT ARE THE plot_method SYNTAX options here? Not seeing those in the documentation, but maybe it's on Rachel's latest round of updates? In her notes, she said "Plot of stat results - option for either volcano or MA plot, heat maps with option to subset to most significant molecules (based on p-value)"
+
+```{r warning=FALSE}
+plot(stats_edger, plot_type = "volcano")
```
Comment (clabornd) [inline vignettes/RNAseq_Processing_Workflow.Rmd:230]: vignette build crashes here due (I think) to a bug in `pmartR::prep_flags`. 
 https://github.com/pmartR/pmartR/blob/241ebe49b6e9afbe96c633629f597d50493e2320/R/plot_fns.R#L5962

It checks for a test type of "EdgeR_LRT", but the statres object has type "EdgeR_F" and so it goes to the wrong logical block and causes problems.  The change was introduced in c1b55e89356a84af67ababf81f40831563e82cde

**Comment by clabornd** [comment]
One more:  `vignettes/Statistical_Analysis_with_pmartR.Rmd` is looking for `modify_mathjax.html` : https://github.com/pmartR/pmartR/blob/241ebe49b6e9afbe96c633629f597d50493e2320/vignettes/Statistical_Analysis_with_pmartR.Rmd#L8 but it appears to have been deleted.

File: vignettes/Data_Normalization.Rmd (hunk 1 above)
Comment (stratkg) [reply vignettes/Data_Normalization.Rmd]: removed showing the head of spans_result

**Comment by stratkg** [comment]
> One more: `vignettes/Statistical_Analysis_with_pmartR.Rmd` is looking for `modify_mathjax.html` :
> 
> https://github.com/pmartR/pmartR/blob/241ebe49b6e9afbe96c633629f597d50493e2320/vignettes/Statistical_Analysis_with_pmartR.Rmd#L8
> 
> but it appears to have been deleted.

added that file back in, must have been removed by accident

File: vignettes/RNAseq_Processing_Workflow.Rmd (hunk 2 above)
Comment (stratkg) [reply vignettes/RNAseq_Processing_Workflow.Rmd:229]: fixed, good catch!

**Review by clabornd — APPROVED** [review]
Looks good to me.  I am still getting some errors related to code style and MSnbase tests, but I think they are okay:

[1] "checking examples ... ERROR\nRunning examples in ‘pmartR-Ex.R’ failed\nThe error most likely occurred in:\n\n> base::assign(\".ptime\", proc.time(), pos = \"CheckExEnv\")\n> ### Name: as.multiData\n> ### Title: Create a 'multiData' object from multiple omicsData objects\n> ### Aliases: as.multiData\n> \n> ### ** Examples\n> \n> \n> library(pmartRdata)\n> \n> # Combine metabolomics and protein object into multidata, both must be log2 \n> # and normalized.\n> mymetab <- edata_transform(omicsData = metab_object, data_scale = \"log2\")\n> mymetab <- normalize_global(omicsData = mymetab, subset_fn = \"all\", norm_fn = \"median\", apply_norm = T)\nError in normalize_global(omicsData = mymetab, subset_fn = \"all\", norm_fn = \"median\",  : \n  T used instead of TRUE\nExecution halted"                                                                                                                                                                              
[2] "checking tests ...\n  Running ‘testthat.R’ [1052s/282s]\n ERROR\nRunning the tests in ‘tests/testthat.R’ failed.\nLast 13 lines of output:\n    3.   │ └─base::asNamespace(ns)\n    4.   └─base::loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]])\n    5.     ├─base::namespaceImportFrom(...)\n    6.     │ └─base::asNamespace(ns)\n    7.     └─base::loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]])\n    8.       ├─base::namespaceImportFrom(...)\n    9.       │ └─base::asNamespace(ns)\n   10.       └─base::loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]])\n   11.         └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)\n   12.           └─base (local) withOneRestart(expr, restarts[[1L]])\n   13.             └─base (local) doWithOneRestart(return(expr), restart)\n  \n  [ FAIL 1 | WARN 12 | SKIP 0 | PASS 2708 ]\n  Error: Test failures\n  Execution halted"
